python/kfserving/kfserving/constants/constants.py

```diff
@@ -18,7 +18,7 @@
 KFSERVING_GROUP = 'serving.kubeflow.org'
 KFSERVING_KIND = 'InferenceService'
 KFSERVING_PLURAL = 'inferenceservices'
-KFSERVING_VERSION = os.environ.get('KFSERVING_VERSION', 'v1alpha2')
+KFSERVING_VERSION = os.environ.get('KFSERVING_VERSION', 'v1beta1')
```

Comment:
Let's still keep the default to be v1alpha2 as all tests are not converted to v1beta1 yet.

Reply:
Will change.